### ISSUE TITLE
FR: SSH wildcard destinations

### ISSUE THREAD

**Issue by legowerewolf** [tailscale/tailscale#4968]
### What are you trying to do?

I have an access rule configured for my network:
```jsonc
                { // Personal devices can access everything
			"Action": "accept",
			"Users":  ["tag:personal"],
			"Ports":  ["*:*"],
		},
```
I want to replicate this in the SSH rules, but they don't support wildcards in the `dst`. 

### How should we solve this?

Support wildcard destinations? 

Alternately, it would be nice if we could configure SSH stuff with the regular ACLs, in the spirit of DRY. Most of my TS network traffic is already SSH. For example:

```jsonc
{ // Personal devices can access everything
    "Action": "accept",
    "Users":  ["tag:personal"],
    "Ports":  ["*:*"],
    "ssh": {
        "users": ["autogroup:nonroot", "root"],     
},
```

You could have even more fun and extend `check` mode to regular ACLs.

### What is the impact of not solving this?

Currently, I've got an ACL tag called `sshable` on all SSH-capable devices and a rule like this:

```jsonc
{
    "action": "accept",
    "src":    ["tag:personal"],
    "dst":    ["tag:sshable"],
    "users":  ["autogroup:nonroot", "root"],
},
```

This won't work nicely for people who don't use ACL tags. I use them for all my devices, to segment my network and prevent less-trustworthy groups of devices from communicating outside their group, so it's not a problem for me.

**Comment by bronson** [comment]
I agree! https://forum.tailscale.com/t/permission-denied-tailscale/2898/7?u=bronson
